doc/config.md

```diff
@@ -0,0 +1,123 @@
+This file supplies the configuration options for both jit-diff and jit-format. The most interesting
+section of the file is the `"default"` section.  Each sub element of `"default"` maps directly to a jit-diff
+or jit-format option name.  Setting a default value here will cause the tools to
+use the given value on start up and then only override that value if new options are passed
+on the command line.
+
+In the jit-diff section, the `"base"` and `"diff"` entries are worth going into
+in more detail.  The `"base"` is set to `"checked_osx-1526"`.  Looking down in the `"tools"` section
+shows that the tool is installed in the `tools` sub-directory of the directory specified by
+`JIT_UTILS_ROOT`.  Any of the tools listed like this can be used in the default section
+as a value, but they can also be passed on the command line
+as the value for `--base` or `--diff`.
+
+Sample config.json:
+```
+{
+  "format": {
+    "default": {
+      "arch": "x64",
+      "build": "Checked",
+      "os": "Windows_NT",
+      "coreclr": "C:\\michelm\\coreclr",
+      "verbose": "true",
+      "fix": "true"
+    }
+  },
+  "asmdiff": {
+    "default": {
+      "base": "checked_osx-1526",
+      "diff": "/Users/russellhadley/Work/dotnet/coreclr/bin/Product/OSX.x64.Checked",
+      "analyze": "true",
+      "frameworksonly": "true",
+      "output": "/Users/russellhadley/Work/dotnet/output",
+      "core_root": "/Users/russellhadley/Work/dotnet/jitutils/fx"
+    },
+    "tools": [
+      {
+        "tag": "checked_osx-1439",
+        "path": "/Users/russellhadley/Work/dotnet/output/tools/checked_osx-1439/Product/OSX.x64.Checked"
+      },
+      {
+        "tag": "checked_osx-1442",
+        "path": "/Users/russellhadley/Work/dotnet/output/tools/checked_osx-1442/Product/OSX.x64.Checked"
+      },
+      {
+        "tag": "checked_osx-1443",
+        "path": "/Users/russellhadley/Work/dotnet/output/tools/checked_osx-1443/Product/OSX.x64.Checked"
+      },
+      {
+        "tag": "checked_osx-1526",
+        "path": "/Users/russellhadley/Work/dotnet/output/tools/checked_osx-1526/Product/OSX.x64.Checked"
+      }
+    ]
+  }
+}
+```
+
+### Listing current defaults
+
+The jit-diff command `list` will read the config.json file in the specified `JIT_UTILS_ROOT` path, and list
+the results.  Adding `--verbose` will show the associated file system paths for installed tools as well.
+
+For example:
+```
+$ jit-diff list
+
+Defaults:
+	base: checked_osx-1526
+	diff: /Users/russellhadley/Work/dotnet/coreclr/bin/Product/OSX.x64.Checked
+	output: /Users/russellhadley/Work/dotnet/output
+	core_root: /Users/russellhadley/Work/dotnet/jitutils/fx
+	analyze: true
+	frameworksonly: true
+
+Installed tools:
+	checked_osx-1439
+	checked_osx-1442
+	checked_osx-1443
+	checked_osx-1526
+```
+
+### Installing new tools
+
+The jit-diff command `install` will download and install a new tool to the default location
+and update the config.json so it can be found.
```

Comment:
I've used install successfully - this is a time saver for me so I'm keen to keep it.